sn_evm/src/wallet.rs

```diff
@@ -0,0 +1,24 @@
+use evmlib::{wallet::{get_random_private_key_for_wallet, Wallet}, Network};
+use evmlib::utils::get_evm_network_from_env;
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+
+pub fn get_random_private_key() -> String {
+    get_random_private_key_for_wallet()
+}
+
+pub fn create_a_evm_wallet(private_key: &String) -> String {
+    let network = get_evm_network_from_env()
+                        .expect("Failed to get EVM network from environment variables");
```

Comment:
`expect` will cause the program to panic/crash. We should return an error here. 
Same at all the places where we use `expect`

Reply:
Agreed too